spacemesh/v2alpha1/smeshing_identities.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+import "google/api/visibility.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+package spacemesh.v2alpha1;
+
+message PoetServicesRequest {}
+
+enum IdentityStatus {
+  STATUS_UNSPECIFIED = 0;
+  STATUS_IS_SYNCING = 1;
+  STATUS_WAIT_FOR_POET_ROUND_START = 2;
+  STATUS_WAIT_FOR_POET_ROUND_END = 3;
+  STATUS_FETCHING_PROOFS = 4;
+  STATUS_POST_PROVING = 5;
+}
+
+
+enum RegistrationStatus {
+  STATUS_UNKNOWN = 0;
+  STATUS_SUCCESS_REG = 1;
+  STATUS_FAILED_REG = 2;
+  STATUS_RESIDUAL_REG = 3;
+  STATUS_NO_REG = 4;
+}
+
+message PoetServicesResponse {
+  message Identity {
+    message PoetInfo {
+      string url = 1;
+      google.protobuf.Timestamp poet_round_end = 2;
+      RegistrationStatus registration_status = 3;
+      string warning = 4;
+    }
+
+    string smesher_id_hex = 1;
```

Comment:
In all other APIs we use `bytes` for the smesher ID. IMHO we should stick to the same format.